scripts/validate-changelog.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+
+import sys
+import requests
+import re
+
+print("Validating that changelog entry is provided in the PR description...")
+
+pr_number = sys.argv[1]
+github_token = sys.argv[2]
+url = "https://api.github.com/repos/mapbox/mapbox-navigation-android/pulls/" + pr_number
+headers = { "accept": "application/vnd.github.v3+json", "authorization": "token " + github_token }
+with requests.get(url, headers = headers) as r:
+  response_json = r.json()
+  pr_description = response_json["body"]
+  pr_labels = response_json["labels"]
+
+  if pr_labels is not None:
+    for label in pr_labels:
+      if label["name"] == "skip changelog":
+        print("`skip changelog` label present, exiting.")
```

Comment:
should we also check case if `skip changelog` exist then <changelog> block shouldn't be present to avoid misunderstanding?

Reply:
Sounds good :+1:

Reply:
Done.